src/Tasks/Resources/Strings.resx

```diff
@@ -267,11 +267,11 @@
     <comment>LOCALIZATION: {0} and {1} are paths.</comment>      
   </data>
   <data name="Copy.Retrying">
-    <value>MSB3026: Could not copy "{0}" to "{1}". Beginning retry {2} in {3}ms. {4}</value>
+    <value>MSB3026: Could not copy "{0}" to "{1}". Beginning retry {2} in {3}ms. {4} {5}</value>
     <comment>{StrBegin="MSB3026: "} LOCALIZATION: {0} and {1} are paths. {2} and {3} are numbers. {4} is an optional localized message.</comment>
```

Comment:
Should we update the comment to say that {5} is either blank or an instantiation of Copy.FileLocked?